### PR TITLE
Migrate confluence docs to README.

### PR DESCRIPTION
**Pull request by relaxnow**
Migrated https://wiki.surfnet.nl/display/stepup/Middleware+Configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ Clone the repository or download the archive to a directory. Install the depende
 
 ## Management API
 
+The configuration of the Gateway should be pushed to the Gateway 
+so that it can be configured from the outside. 
+This is done by making a POST request to an URL at the middleware, 
+with the new configuration as JSON in the request body.
```
Comment (rjkip) [inline README.md]: The Middleware also has configuration managed by this API. What about:

> The Middleware and Gateway applications have various run-time configuration values that are managed through a JSON HTTP API served by the Middleware application. This section is intended to document the requirements and structure of the configuration.

Comment (rjkip) [inline README.md]: Ah yes, sorry, this is from the wiki. Do with it what you want :p

**Comment by rjkip** [comment]
Nice to have in here :+1: (maybe time to split off into separate documents :yum:)

File: README.md [hunk 2]
```diff
@@ -103,6 +135,222 @@ curl -XPOST -v \
...
+			<td colspan="1">ras</td>
+			<td colspan="1">array</td>
+			<td colspan="1">
+			<table border="0" cellpadding="0" cellspacing="0">
+				<tbody>
+					<tr>
+						<td>
+						<p><code>[</code></p>
+						<p><code>&nbsp;&nbsp;&nbsp;&nbsp;</code><code>[</code></p>
+						<p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</code><code>&#39;commonName&#39;</code>&nbsp;<code>=&gt;&nbsp;</code><code>&#39;Jan Modaal&#39;</code><code>,</code></p>
```
Comment (arothuis) [inline README.md]: Nit: There is a comma in this array, yet there are no comma's after the other array entries.

**Pull request merged**